eureka-client/src/main/java/com/netflix/discovery/DiscoveryClient.java

```diff
@@ -196,6 +196,30 @@
 
         @Inject(optional = true)
         private Provider<HealthCheckHandler> healthCheckHandlerProvider;
+
+        public EventBus getEventBus() {
```

Comment:
Hmm, adding a new constructor to `DiscoveryClient` would seem more invasive than needs be.  Let me update and see what you think.